jobs/integration/validation.py

```diff
@@ -2524,8 +2525,13 @@ async def ceph_apps(model, tools):
     elif series_idx > SERIES_ORDER.index("jammy"):
         pytest.fail("ceph_charm_channel is undefined past jammy")
 
-    if any(a in model.applications for a in ["ceph-mon", "ceph-osd"]):
-        pytest.skip("Skipped since ceph apps are already installed")
+    all_apps = ["ceph-mon", "ceph-osd", "ceph-fs"]
+    if any(a in model.applications for a in all_apps):
+        # allow currently deployed ceph apps to run tests
```

Comment:
hmm, i'm worried about this.  i have a strong remembrance of skipping pre-existing ceph deployment testing.  as in, if `validation.py` didn't deploy it, we don't want to test it since that could affect other products relying on this model.

Perhaps it's ok as we're yielding / returning here and won't fall into the ceph removal below, but i'd like a +2 here in case we need to be careful about running [validate_storage_class](https://github.com/charmed-kubernetes/jenkins/blob/f1e382ac3862c386a2905950ebaa6b15e3f23b76/jobs/integration/utils.py#L366) on a preexisting ceph.

@Cynerva, wadda you think?

Reply:
i can remove it -- i had it in there to be able to run the tests over and over without the ceph setup/teardown

Reply:
or I could add a test option like `--use-existing-ceph-apps` defaulted to `False`

I can run my tests -- but the normal runs wouldn't be touched

Reply:
This seems fine to me. If Ceph is in the same model as CK then I think it's fair to assume that they're supposed to be integrated. And if they are, then when I run validation, I would expect that integration to be tested.

Reply:
ok, i'm gonna take that as a LGTM...